api/v1/controllers/rooms.js

```diff
@@ -48,7 +51,23 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   findRooms(req, res, next) {
-    doFind(req, res, next, helper);
+    const type = req.swagger.params.type;
+    if (type && !u.looksLikeId(type.value)) {
+      RoomType.findOne({
+        where: {
+          name: { [Op.iLike]: req.swagger.params.type.value },
+        },
+      })
+      .then((o) => {
+        if (o) {
+          req.swagger.params.type.value = o.dataValues.id;
```

Comment:
if it's by type, does the SQL return just one RoomType? 

"o" is just one?

Reply:
@cassiodias we want it to just return 1 roomType. We're filtering rooms by their roomType and There should not be more than 1 roomType with the same name.